modules/apps/headless-apio/structure/structure-apio-api/src/main/java/com/liferay/structure/apio/architect/util/StructureFieldConverter.java

```diff
@@ -0,0 +1,42 @@
+/**
```

Comment:
Brian usually likes *Helper or *Util classes (util for static, helper for components) but let's see if it passes :)